python/src/tests/test_consume_message.py

```diff
@@ -122,24 +122,69 @@ def test_request_with_expired_timeout_is_discarded(self):
 
             assert result is None
 
-    # These tests only work locally with inframock running. Keeping in case
-    # we want to mock to be able to run these tests.
-    #
-    # def test_consume_request_with_non_expired_timeout_successfully(self):
-    #     request = {
-    #         "experimentId": "random-experiment-id",
-    #         "timeout": "2900-01-01 00:00:00",
-    #         "uuid": "random-uuid",
-    #         "ETag": "random-etag",
-    #     }
-
-    #     with mock.patch("worker.consume_message._read_sqs_message") as m:
-    #         m.return_value = request
-    #         result = consume()
-
-    #         assert result == {
-    #             "experimentId": "random-experiment-id",
-    #             "timeout": "2900-01-01 00:00:00",
-    #             "uuid": "random-uuid",
-    #             "ETag": "random-etag",
-    #         }
+    def test_consume_request_non_expired_timeout_results_not_in_s3(self):
```

Comment:
the reason why this test was failing was an unmocked call to S3 that I have now fixed with stubbers. Similar idea like in the other tests above